handbook/product/README.md

```diff
@@ -129,6 +129,8 @@ Product Managers [write user stories](https://fleetdm.com/handbook/company/devel
 ## Drafting user stories
 Product Designers [draft user stories](https://fleetdm.com/handbook/company/development-groups#drafting) that have been prioritized by PMs. If the estimated user stories for a product group exceed [that group's capacity](https://fleetdm.com/handbook/company/product-groups#current-product-groups), all new design work for that group is paused, and the designer will contribute in other ways (documentation & handbook work, Figma maintenance, QA, etc.) until the PM deprioritizes estimated stories to make room, or until the next sprint begins. (If the designer has existing work-in-progress, they will continue to review and iterate on those designs and see the stories through to estimation.)
 
+In cases where a designer needs more clarity on the requirements of a user story, they will place the issue ticket into the "needs clarity" column of the drafting board, assign the group product manager, and communicate the areas that require additional information. 
```

Comment:
~In cases where a designer needs more clarity on the requirements of a user story~ => User stories are clearly stated and easy to read. When user stories are unclear or don't match the discussed end result they will be placed into the "Needs clarity" column of the drafting board. The group product manager is then reassigned to rework the user story.

Reply:
@Sampfluger88 changes made.

Reply:
If an issue's title or user story summary (_"as a…I want to…so that"_) does not match the intended change being discussed, the designer will move the issue to the "Needs clarity" column of the drafting board and assign the group product manager.  The group product manager will revisit ASAP and edit the issue title and user story summary, then reassign the designer and move the issue back to the "Prioritized" column.